### DISCUSSION TITLE
How to solve the smtp concurrency limit problem?

### DISCUSSION THREAD

**Discussion by YaoJusheng** [apache/airflow#24314] [Q&A]
When I execute the dag, I found that the emailoperator has a concurrency limit error, 
such as:
```
SMTPDataError: (432, b'4.3.2 Concurrent connections limit exceeded.
```
How to fix it?

**Comment by potiuk** [comment]
On Airlfow side - use pools. On server side - increase the number of parallel connections allowed.

**Reply by YaoJusheng** [reply]
I am using EmailOperator in Airlfow, so how should I configure the connection pool?

**Reply by potiuk** [reply]
Use "pool" functionality of Airlfow. Look it up in Airlfow docs.

**Reply by YaoJusheng** [reply]
Okay, I have solved it. Thanks.
